attest/tpm_windows.go

```diff
@@ -190,6 +190,9 @@ func (t *windowsTPM) eks() ([]EK, error) {
 	if i.Manufacturer.String() == manufacturerIntel {
 		ek.CertificateURL = intelEKURL(pub)
 	}
+	if i.Manufacturer.String() == manufacturerAMD {
```

Comment:
There is duplication between this and wrapped_tpm20.go and repetition with multiple if statements.

What about moving this logic into tpm.go into a helper function ekCertUrl() and using a switch statement to more efficiently cover the multiple cases?